webrtc.html

```diff
@@ -8097,6 +8097,17 @@ <h2>
                           exception/throw =] a {{RangeError}}.
                         </p>
                       </li>
+                      <li>
+                        <p>
+                          Verify that the value of each
+                          {{RTCRtpEncodingParameters/maxFramerate}
+                          member in <var>sendEncodings</var> that is defined
```

Comment:
```suggestion
                          member in <var>sendEncodings</var> that is present
```

Reply:
The text for `scaleResolutionDownBy` uses `defined` and I'll keep that for consistency, feel free to update both if you want later though.